go.mod

```diff
@@ -5,9 +5,50 @@ go 1.16
 require (
 	github.com/onsi/ginkgo v1.14.1
 	github.com/onsi/gomega v1.10.2
-	k8s.io/api v0.20.2
-	k8s.io/apimachinery v0.20.2
-	k8s.io/client-go v0.20.2
+	github.com/openshift/library-go v0.0.0-20210727125124-5379485020aa
+	k8s.io/api v0.21.3
+	k8s.io/apimachinery v0.21.3
+	k8s.io/client-go v0.21.3
+	k8s.io/kube-aggregator v0.21.3
 	sigs.k8s.io/controller-runtime v0.8.3
 	sigs.k8s.io/yaml v1.2.0
+
 )
+
+// Pinned to k8s v0.20.0
```

Comment:
What is the reason for pinning to this k8s version?

Reply:
I used this k8s version as it is a recent version which is also compatible with the version of controller-runtime we are on. Also it is the one we use in MCH Operator repository.

This import is necessary for parsing certain resource types.